components/NavBarContainer.js

```diff
@@ -41,22 +40,51 @@ var NavBarContainer = React.createClass({
 
   // We render both the current and the previous navbar (for animation)
   render: function() {
-    if(this.props.currentRoute.trans === true)
-      var trans = {backgroundColor: 'transparent'}
-    else
-      var trans = {}
+    var trans,
+      navbarStyle,
+      navbarContent;
 
-    return (
-      <View style={[styles.navbarContainer, this.props.style, trans]}>
-        {this.props.currentRoute.trans
-          ?( <View /> )
-          :( <NavBarContent
-            route={this.state.previousRoute}
-            backButtonComponent={this.props.backButtonComponent}
-            rightCorner={this.props.rightCorner}
-            titleStyle={this.props.titleStyle}
-            willDisappear="true" /> )
-        }
+    if (this.props.currentRoute.trans) {
+      trans = {backgroundColor: 'transparent'};
+    }
+    else {
```

Comment:
This should probably be consistent with the One True Brace in the rest of the code.